SC_redesign/Crop_and_Soil/field/field.py

```diff
@@ -355,17 +356,27 @@ def _execute_manure_application(self, requested_nitrogen: float, requested_phosp
             Mass of phosphorus requested to be in this manure application (kg)
         field_coverage : float
             Fraction of the field this manure is applied to (unitless)
+        application_depth : float
+            Depth at which fertilizer is injected into the soil (mm).
+        surface_remainder_fraction : float
+            Fraction of fertilizer applied that remains on the soil surface after application (unitless).
         year : int
             Calendar year in which this manure application occurs.
         day : int
             Julian day on which this manure application occurs.
 
         Notes
         -----
-        Because potassium is not currently specified in the manure request results, it is recorded as None.
+        Because potassium is not currently specified in the manure request results, it is recorded as None. This method
+        also checks for invalid application depths and surface remainder fractions. If invalid values are found, they
+        are corrected, an error is raised to the OutputManager, and execution continues with the new values.
```

Comment:
Not sure if this is an important distinction but wanted the note to be a little clearer about what happens if invalid values are found - that the simulation won't be interrupted by the normal error/exception process, we're just logging it to the OM.
```suggestion
        are corrected, an error is logged to the OutputManager, and execution continues with the new values.
```

Reply:
Good call, that is an important distinction to make.